samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,99 @@ nRF5340: Network core bootloader
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpunet
 
 .. _net_bootloader_build_and_run:
 
 Building and running
 ********************
 
-The source code of the sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
+The recommended way of building the sample is to use the :ref:`multi-image feature <ug_multi_image>` of the build system.
 
-The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
-This sample is included automatically if the application in the nRF5340 network core has the :option:`CONFIG_SECURE_BOOT` option set.
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+You can include the sample in your multi-image build in two different ways:
 
-A new set of firmware update files is generated by the build system when this image is included in a multi-image build.
-These files are identical to those described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+* Enabling the :option:`CONFIG_SECURE_BOOT` option in the application that runs on the nRF5340 network core.
+  With this option enabled, the build system includes the sample automatically.
+* Enabling the :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53` options in the application that runs on the application core.
+  In a multi-image build where there is also an application running on the network core, enabling MCUboot in the application core automatically includes the network core bootloader.
+  As such, with these options enabled, the build system automatically enables the :option:`CONFIG_SECURE_BOOT` option for the network core application.
 
-Testing
-=======
+  In addition, you must also enable the :ref:`subsys_pcd` library for the MCUboot image, enabling the option :option:`CONFIG_PCD`.
 
-To test the network core bootloader sample run the following commands:
+  .. note::
 
-#. |connect_terminal|
+     The firmware is signed exactly like the application firmware is signed.
+     As such, the firmware is also verified by MCUboot on the application core.
+     See :ref:`subsys_pcd` for more details.
 
-   Note that on the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+When you include this image in a multi-image build, the build system generates a new set of firmware update files
+These files match the ones described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
 
-#. ``west build -b nrf5340dk_nrf5340_cpuapp -d build_netboot samples/bluetooth/peripheral_uart -f -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_PCD=y``
```

Comment:
I like how concise these changes are, but I would suggest mentioning the `samples/bluetooth/peripheral_uart` sample as the basis for the multi-image build for testing this. It's a great sample to use for netboot, and useful to know about for users that are just starting to get familiar with netboot on the 5340.

Reply:
@kyberdin added a mention in a note